dht/i_partitioner.hh

```diff
@@ -24,6 +24,22 @@
 
 namespace dht {
 
+// Returns a successor for token_range tr, capped at maximum_token.
```

Comment:
> to implement a stateless sstable set that doesn't
> need to be refreshed every time any of the

You're breaking a strong invariant here. How does it affect the rest of the system?

Reply:
/cc @tgrabiec